tools/cargo-miden/src/lib.rs

```diff
@@ -240,6 +240,7 @@ where
                 )
                 .await
             })?;
+            dbg!(&wasm_outputs);
```

Comment:
*Nit:* probably didn't mean to leave this in here

Reply:
Thanks for spotting this!